docs/site/src/components/AppRouter.js

```diff
@@ -46,7 +46,11 @@ export default function AppRouter() {
       history={hashHistory}
       render={applyRouterMiddleware(useScroll())}
     >
-      <Route title="Material UI" path="/" component={AppFrame}>
+      <Route
+        title="Material UI"
+        path="/"
+        component={(props) => <AppFrame demos={demos} apiDocs={apiDocs} {...props} />}
```

Comment:
Well, having added an intermediate component to contain the logic, it starts to make more sense to use context.